.gitignore

```diff
@@ -2,3 +2,4 @@ target/
 /.sccache/
 .DS_Store
 .soroban
+errors/test_snapshots/
```

Comment:
Should we ignore all test snapshot directories?

Reply:
Yup with `**/test_snapshots/`.